plugin/trino-hive/src/main/java/io/trino/plugin/hive/coercions/CoercionUtils.java

```diff
@@ -131,6 +131,9 @@ public static Type createTypeFromCoercer(TypeManager typeManager, HiveType fromH
         if (fromType instanceof BooleanType && toType instanceof VarcharType toVarcharType) {
```

Comment:
Document in `hive.md` https://trino.io/docs/current/connector/hive.html#schema-evolution the new schema evolution scenario